pyproject.toml

```diff
@@ -0,0 +1,155 @@
+[project]
+name = "sqlmesh"
+dynamic = ["version"]
+description = "Next-generation data transformation framework"
+readme = "README.md"
+authors = [{ name = "TobikoData Inc.", email = "engineering@tobikodata.com" }]
+license = { file = "LICENSE" }
+requires-python = ">= 3.9"
+dependencies = [
+    "astor",
+    "click",
+    "croniter",
+    "duckdb!=0.10.3",
+    "dateparser",
+    "hyperscript>=0.1.0",
+    "importlib-metadata; python_version<'3.12'",
+    "ipywidgets",
+    "jinja2",
+    "pandas",
+    "pydantic>=2.0.0",
+    "requests",
+    "rich[jupyter]",
+    "ruamel.yaml",
+    "sqlglot[rs]~=26.8.0",
+    "tenacity",
+    "time-machine",
```

Comment:
Shouldn't this be a dev dependency?

Reply:
It's used in the testing module: https://github.com/TobikoData/sqlmesh/blob/e100fbf126761a86004fd50d235798e64977c562/sqlmesh/core/test/definition.py#L678-L680